lib/lexisnexis/threat_metrix/verification_request.rb

```diff
@@ -0,0 +1,83 @@
+require 'base64'
+require 'typhoeus'
+require 'uri'
+require 'digest'
+
+module LexisNexis
+  module ThreatMetrix
+    class VerificationRequest
```

Comment:
I decided not to subclass the base `Request` in this gem because I think it's all easier to read when it's in one file ... and there's not that much to copy